cmd/wstail/tail.go

```diff
@@ -288,7 +249,6 @@ func (r *processGroup) Close() (err error) {
 	for _, closer := range r.closers {
 		closer.Close()
 	}
-	r.terminate()
```

Comment:
We will give error `wait: no child processes` because process exited cleanly before we call wait.